src/platform/forms/sass/_m-schemaform.scss

```diff
@@ -535,18 +535,14 @@ legend.schemaform-label.schemaform-file-label {
   }
 }
 
+/* Highlight accordion item with content that needs attention */
 va-accordion-item[data-unviewed-pages="true"] {
-  --item-border: 3px solid var(--color-secondary-lightest) !important;
-
   &::part(accordion-header) {
-    background-color: var(--color-secondary-lightest);
-    border-color: var(--color-secondary-lightest);
+    background-color: #{$color-secondary-lightest};
+  }
+  &::part(accordion-content) {
+    border-color: #{$color-secondary-lightest};
   }
-}
-
-.form-review-panel .accordion-header > button.usa-button-unstyled {
-  padding: 2.5rem 4.5rem 2.5rem 2.5rem;
-  line-height: 2.2rem;
 }
```

Comment:
`.accordion-header` existed in the HTML accordion (non-web component) version

Reply:
ah nice. Thanks for that context.